packages/cms-cli/commands/logs.js

```diff
@@ -121,14 +114,17 @@ const tailLogs = async ({
   tail(initialAfter);
 };
 
-const action = async ({ functionPath }, options) => {
+exports.command = 'logs <path>';
+exports.describe = 'get logs for a function';
+
+exports.handler = async options => {
   loadAndValidateOptions(options);
 
-  const { latest, file, follow, compact } = options;
+  const { latest, tail, compact, path: functionPath } = options;
```

Comment:
I had set this to `tail` initially and after discussion we agreed on `follow`. Also this would be a breaking change, right?

Reply:
I have it aliased so it won't break, but sounds good. I'll swap it back and make tail the alias.

```
yargs.option('tail', {
    alias: ['t', 'follow', 'f'],
    describe: 'tail logs',
    type: 'boolean',
  });
```